lib/shared/puppet_forge/error.rb

```diff
@@ -26,5 +26,18 @@ def multiline
     end
   end
 
+  class SymlinkInModuleError < PuppetForge::Error
```

Comment:
No, it isn't. This was left over from a previous mechanism for moving the symlink files back out. Removing it.